python/triton/language/core.py

```diff
@@ -2528,9 +2528,12 @@ def kernel(...):
         kernel argument.  The kernel argument only pipelines loads that feed
         into :code:`dot` operations, while this attribute tries to pipeline most
         (though not all) loads in this loop.
+    :param unrolled_iteration: Tells the Triton IR level loop unroller how many
+        times to unroll a for loop that this range is used with. Less than 2 for
+        this value implies no unrolling. For now we only support even unrolling.
     """
 
-    def __init__(self, arg1, arg2=None, step=None, num_stages=None):
+    def __init__(self, arg1, arg2=None, step=None, num_stages=None, unrolled_iteration=1):
```

Comment:
I'd suggest to use `None` as the default value to reduce the dependency on the backend implementation.